src/plugins/lightscript.js

```diff
@@ -134,40 +134,45 @@ pp.parseObjectComprehension = function(node) {
   return this.finishNode(node, "ObjectComprehension");
 };
 
-// c/p parseBlock
+pp.finishWhiteBlock = function(node) {
+  if (!node.body.length) {
+    this.unexpected(node.start, "Expected an Indent or Statement");
+  }
+
+  this.addExtra(node, "curly", false);
+  return this.finishNode(node, "BlockStatement");
+};
 
-pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
+pp.parseInlineWhiteBlock = function(node) {
+  if (this.state.type.startsExpr) return this.parseMaybeAssign();
+  // oneline statement case
+  node.body = [this.parseStatement(true)];
```

Comment:
or I guess you're saying just return the parsed statement and don't wrap it at all?